packages/twenty-server/src/database/commands/upgrade-version/0-42/0-42-standardization-of-actor-composite-context-type.ts

```diff
@@ -0,0 +1,112 @@
+import { InjectRepository } from '@nestjs/typeorm';
+
+import chalk from 'chalk';
+import { Command } from 'nest-commander';
+import { FieldMetadataType } from 'twenty-shared';
+import { IsNull, Repository } from 'typeorm';
+
+import {
+  ActiveWorkspacesCommandOptions,
+  ActiveWorkspacesCommandRunner,
+} from 'src/database/commands/active-workspaces.command';
+import { CommandLogger } from 'src/database/commands/logger';
+import { Workspace } from 'src/engine/core-modules/workspace/workspace.entity';
+import { FieldMetadataEntity } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+import { WorkspaceMetadataVersionService } from 'src/engine/metadata-modules/workspace-metadata-version/services/workspace-metadata-version.service';
+import { TwentyORMGlobalManager } from 'src/engine/twenty-orm/twenty-orm-global.manager';
+
+@Command({
+  name: 'upgrade-0.42:standardization-of-actor-composite-context-type',
+  description: 'Add context to actor composite type.',
+})
+export class StandardizationOfActorCompositeContextTypeCommand extends ActiveWorkspacesCommandRunner {
+  protected readonly logger;
+
+  constructor(
+    @InjectRepository(Workspace, 'core')
+    protected readonly workspaceRepository: Repository<Workspace>,
+    private readonly twentyORMGlobalManager: TwentyORMGlobalManager,
+    @InjectRepository(FieldMetadataEntity, 'metadata')
+    private readonly fieldMetadataRepository: Repository<FieldMetadataEntity>,
+    private readonly workspaceMetadataVersionService: WorkspaceMetadataVersionService,
+  ) {
+    super(workspaceRepository);
+    this.logger = new CommandLogger({
+      constructorName: this.constructor.name,
+      verbose: false,
+    });
+    this.logger.setVerbose(false);
+  }
+
+  async executeActiveWorkspacesCommand(
+    _passedParam: string[],
+    options: ActiveWorkspacesCommandOptions,
+    workspaceIds: string[],
+  ): Promise<void> {
+    this.logger.log(`Running add-context-to-actor-composite-type command`);
+
+    if (options?.dryRun) {
+      this.logger.log(chalk.yellow('Dry run mode: No changes will be applied'));
+    }
+
+    for (const workspaceId of workspaceIds) {
+      try {
+        await this.execute(workspaceId, options?.dryRun);
+        this.logger.verbose(`Added for workspace: ${workspaceId}`);
```

Comment:
let's add a counter to know where we are compared to the totalCount of worksapce too